packages/suite/src/middlewares/wallet/blockchainMiddleware.ts

```diff
@@ -15,6 +16,7 @@ const blockchainMiddleware =
         switch (action.type) {
             case BLOCKCHAIN.CONNECT:
                 api.dispatch(blockchainActions.onConnect(action.payload.coin.shortcut));
+                api.dispatch(cardanoStakingActions.fetchTrezorPools());
```

Comment:
this call takes place even in case when cardano is disabled. I would rather limit it only for cases when cardano is active. would there be problem to trigger this call from some useEffect that is guaranteed to be rendered only in cardano context maybe?